### PR TITLE
Add cromwell labels

### PR DESCRIPTION
**Pull request by vivbak**
Add custom labels to cromwell instance as well as standard compute-category label.

### CODE REVIEW

File: analysis_runner/cromwell.py [hunk 1]
```diff
@@ -151,6 +160,11 @@ def run_cromwell(
                 workflow_input_prefix + k: v for k, v in _inputs_dict.items()
             }
 
+    if labels:
+        _labels = json.loads(labels)
+    if labels is None:
+        _labels = None
```
Comment (illusional) [inline analysis_runner/cromwell.py:166]:
```suggestion
    _labels = None
    if labels:
        _labels = json.loads(labels)
```

File: analysis_runner/dataproc.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def hail_dataproc_job(
 
     if labels is None:
         labels = {}
-    labels.update({'compute_category': 'dataproc'})
+    labels.update({'compute-category': 'dataproc'})
```
Comment (illusional) [inline analysis_runner/dataproc.py:53]:
```suggestion
    labels['compute-category'] = 'dataproc'
```

File: server/cromwell.py [hunk 3]
```diff
@@ -199,12 +200,18 @@ async def cromwell(request):
                 )
 
             cromwell_post_url = CROMWELL_URL + '/api/workflows/v1'
+
+            google_labels = {'compute-category': 'cromwell'}
+            if labels:
+                google_labels.update(labels)
```
Comment (illusional) [inline server/cromwell.py:206]: It's probably fine, but this does allow the user to override `compute-category` to be whatever they want. You could reorder this if that's unintended behaviour.

**Review by illusional — APPROVED** [review]
Awesome work!

File: server/cromwell.py (hunk 3 above)
Comment (vivbak) [reply server/cromwell.py:207]: Great point, thanks! I've swapped the order.

**Pull request merged**